### PR TITLE
[IOTDB-974] Convert tables from HTML language to markdown grammar in all md files

### PR DESCRIPTION
**Pull request by JesseZhou-1**
I rewrote most of the tables from HTML language to markdown grammar, and used figures to represent the complex ones. If there is any table in HTML language that I missed, please let me know. I will make sure to convert them afterward.

### CODE REVIEW

**Comment by jixuan1989** [comment]
Thanks @JesseZhou-1 . As the master branch merged a PR yesterday and the Memory Estimation Tool.md is deleted in that PR, you have to solve the conflict (just remove the file).

**Comment by JesseZhou-1** [comment]
@jixuan1989 Done

File: docs/UserGuide/Ecosystem Integration/MapReduce TsFile.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ With this connector, you can
 ## System Requirements
 
 |Hadoop Version | Java Version | TsFile Version|
-|-------------  | ------------ |------------ |
+|:------------- : | :------------ :|:------------: |
```
Comment (jixuan1989) [inline docs/UserGuide/Ecosystem Integration/MapReduce TsFile.md:49]: something error here.
maybe we need "blank" between ":" and "|"

**Review by jixuan1989 — COMMENTED** [review]
And one more issue:  seems the first and second columns are wider than other columns in all png files... If we can tune it, it will be better.

File: docs/UserGuide/Ecosystem Integration/MapReduce TsFile.md (hunk 1 above)
Comment (JesseZhou-1) [reply docs/UserGuide/Ecosystem Integration/MapReduce TsFile.md]: solved

**Comment by JesseZhou-1** [comment]
update all png files, all columns now have the exact same width.

**Review by qiaojialin — APPROVED** [review]
Good job!

**Pull request merged**